core/main.go

```diff
@@ -135,6 +138,14 @@ func InitRepo(conf InitConfig) error {
 		return ErrAccountRequired
 	}
 
+	logLevels := map[string]string{}
+	if conf.Debug {
+		logLevels = getTextileDebugLevels()
+	}
+	if _, err := setLogLevels(conf.RepoPath, logLevels, conf.LogToDisk); err != nil {
```

Comment:
Unfortunately, I had to take this method off the node again because we need to use it before the `Textile` is created, here in init, but also in `NewTextile`, before it's finished being created (need to see debug output from migrations, for example)